motr/m0kv/index_op.c

```diff
@@ -191,6 +209,10 @@ static int index_op(struct m0_realm    *parent,
 
 	m0_fid_tassume(fid, &m0_dix_fid_type);
 	m0_idx_init(&idx, parent, (struct m0_uint128 *)fid);
+
+	if (is_enf_meta)
```

Comment:
Do I understand correctly that for normal operations there are only two scenarios:
 - both -e and -v <fid> is specified
 - no -e and -v specified

In this case for normal operations -v becomes mandatory when -e is specified, otherwise no pver info will be written to `idx->in_attr` but operation will be executed. Need to check consistency of passing of -e and -v somewhere.

Reply:
Here three scenarios not two:
1. Without -e and -v
2. with -e for index create only
3. -e and -v for PUT/GET/DEL/NEXT operations.

We will discuss if you have any queries.

Reply:
-v becomes mandatory when -e is specified for PUT/GET/NEXT/DEL operations.

Reply:
Added more validation for pool version.